### DISCUSSION TITLE
How to set color or brightness of inactive pane?

### DISCUSSION THREAD

**Discussion by Jaehaks** [microsoft/terminal#17624] [Q&A] [answered]
I cannot recognize what the inactive pane is.
There are settings to change cursor shape, but I think it is not enough.

**Comment by zadjii-msft** [comment, marked as answer]
Did you try setting an "unfocused appearance" in the settings?

You an try something like: 

```json
                "unfocusedAppearance": 
                {
                    "background": "#222222",
                    "colorScheme": "Solarized Light",
                    "opacity": 50
                },
```

**Reply by Jaehaks** [reply]
thank you for your reply:)
unfortunately, it seems that there are no feature to adjust all brightness to make more dark unfocused pane. but it is good to change opacity and background
